docs/docs/developers/development/page-editor/extending-editor/index.md

```diff
@@ -0,0 +1,116 @@
+# Extending Page Editor
+
+## Overview
+
+Developers may need to extend functionality of _Page editor_. It can be done by providing _Editor extension_.
+_Editor extension_ is delivered as JavaScript and use _Page editor_ API.
+
+!!! Info "Note"
+        Detailed specification of _Page editor_ API will be provided together with next versions of WebSight CMS.
```

Comment:
@pawel-boguski-ds, do we mean the very next release 1.9.0, or just sooner or later?

Reply:
Sooner or later.